doc/Manuals/ug-sec_x_road_security_hardening.md

```diff
@@ -0,0 +1,165 @@
+
+## Table of Contents <!-- omit in toc -->
+<!-- toc -->
+
+* [License](#license)
+* [1. Introduction](#1-introduction)
+    * [1.1 Target Audience](#11-target-audience)
+    * [1.2 Terms and abbreviations](#12-terms-and-abbreviations)
+    * [1.3 References](#13-references)
+* [2. User Management](#2-user-management)
+    * [2.1 Configuring account lockout](#21-configuring-account-lockout)
+        * [2.1.1 Considerations and risks](#211-considerations-and-risks)
+        * [2.1.2 Account lockout examples](#212-account-lockout-examples)
+    * [2.2 Configuring password policies](#22-configuring-password-policies)
+        * [2.2.1 Considerations and risks](#221-considerations-and-risks)
+    * [2.3 Ensuring User Account Security](#23-ensuring-user-account-security)
+* [3.Admin UI (Central Server and Security Server)](#3-admin-ui-central-server-and-security-server)
+    * [3.1 Host header injection mitigation](#31-host-header-injection-mitigation)
+  
+<!-- tocstop -->
+
+## License
+
+This document is licensed under the Creative Commons Attribution-ShareAlike 3.0 Unported License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/3.0/.
+
+# 1. Introduction
+
+You may want to harden the security of your X-Road instance by configuring additional security policies within your X-Road infrastructure.
+The security measures that are introduced in this guide are common security policies that can be configured on operating system level.
+
+## 1.1 Target Audience
+
+The intended audience of this User Guide are X-Road administrators (Central or Security server) who are responsible for X-Road instance set-up and/or everyday management of the X-Road infrastructure.
+
+## 1.2 Terms and abbreviations
+
+See X-Road terms and abbreviations documentation \[[TA-TERMS](#Ref_TERMS)\].
+
+## 1.3 References
+
+1. <a id="Ref_IG-CS" class="anchor"></a>\[IG-CS\] X-Road 7. Central Server Installation Guide. Document ID: [IG-CS](ig-cs_x-road_6_central_server_installation_guide.md).
+2. <a id="Ref_UG-CS" class="anchor"></a>\[UG-CS\] X-Road 7. Central Server User Guide. Document ID: [UG-CS](ug-cs_x-road_6_central_server_user_guide.md).
+3. <a id="Ref_IG-SS" class="anchor"></a>\[IG-SS\] X-Road: Security Server Installation Guide. Document ID: [IG-SS](ig-ss_x-road_v6_security_server_installation_guide.md).
+4. <a id="Ref_UG-SS" class="anchor"></a>\[UG-SS\] X-Road 7. Security Server User Guide. Document ID: [UG-SS](ug-ss_x-road_6_security_server_user_guide.md).
+5. <a id="Ref_TERMS" class="anchor"></a>\[TA-TERMS\] X-Road Terms and Abbreviations. Document ID: [TA-TERMS](../terms_x-road_docs.md).
+6. <a id="Ref_UG-SYSPAR" class="anchor"></a>\[UG-SYSPAR\] X-Road: System Parameters User Guide. Document ID: [UG-SYSPAR](ug-syspar_x-road_v6_system_parameters.md).
```

Comment:
Let's change them so they all start with an `X-Road:` prefix rather than `X-Road 7.`. In that way, it is more uniform, and I don't see a need to include the version number since we version the documentation as well and it should always correspond to the same version anyway.